### PR TITLE
Notes to catalogue item #264

### LINKED ISSUE #264: Add notes to the catalogue item
- [ ] Add notes to catalogue item 
- [ ] display notes in the items dialog (This maybe useful)

### PR DESCRIPTION
**Pull request by MatteoGuarnaccia5**
## Description

Implement notes for catalogue items. Added tooltip to item dialog as well which shows the parent catalogue item note.

## Testing instructions

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
## Agile board tracking

closes #264

### CODE REVIEW

**Comment by joshuadkitenge** [comment]
> Just some minor things.
> 
> @joshuadkitenge I noticed we allow multiline notes in text boxes, but always condense them into one line when displaying them elsewhere, I think its fine as it is for now but thought I should mention it - I think it would be tricky for tooltips anyway.

Is this going to be a problem for the normal text being rendered in the table (if it is too long) ?   @joelvdavies

File: src/items/itemDialog.component.test.tsx [hunk 1]
```diff
@@ -431,6 +431,30 @@ describe('ItemDialog', () => {
         warranty_end_date: '2023-05-18T23:00:00.000Z',
       });
     }, 10000);
+
+    it('Displays catalogue item notes tooltip on hover', async () => {
+      createView();
+      await waitFor(() => {
+        expect(
+          screen.getByLabelText('Catalogue item note: null')
```
[the remark this answers is not in the document]
Comment (joshuadkitenge) [reply src/items/itemDialog.component.test.tsx:439]: Yeah I think for now it simplest to return "None" ('Catalogue item note: None').

**Comment by joshuadkitenge** [comment]
> > > Just some minor things.
> > > @joshuadkitenge I noticed we allow multiline notes in text boxes, but always condense them into one line when displaying them elsewhere, I think its fine as it is for now but thought I should mention it - I think it would be tricky for tooltips anyway.
> > 
> > 
> > Is this going to be a problem for the normal text being rendered in the table (if it is too long) ? @joelvdavies
> 
> Yes we don't prevent long names and replace them with tooltips anywhere so technically they could all display tonnes of text (not that I would expect extremely long fields everywhere), I only meant the notes field in the comment as you can write a note like
> 
> ```
> Some
> 
> notes
> ```
> 
> Which then gets displayed as `Some notes` on the landing page/in the tooltips.

Adding this` whiteSpace: 'pre-line'` to the sx prop should sort out that problem.

**Review by joshuadkitenge — CHANGES_REQUESTED** [review]
Works well. Could you please add whiteSpace: 'pre-line' for all descriptions and notes (obsolete reason) displayed throughout the website any other multiline content

**Review by joshuadkitenge — APPROVED** [review]

**Review by joelvdavies — APPROVED** [review]

**Pull request merged**